app/src/panels/AddEmployee.js

```diff
@@ -62,7 +64,7 @@ function AddEmployeePanelContent({
   const [address, setAddress] = useState('')
   const [role, setRole] = useState('')
   const [salary, setSalary] = useState('')
-  const [startDate, setStartDate] = useState(dayjs())
+  const [startDate, setStartDate] = useState(dayjs().toDate())
```

Comment:
Ah maybe here we can do just `new Date()` 

```suggestion
  const [startDate, setStartDate] = useState(new Date())
```

Reply:
I think this should be just null, so the user has to choose a day instead of making today's date as default

Reply:
I think is better!